.changeset/forty-worms-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix: svelte-kit sync gracefully handles a nonexistent src/routes folder
```

Comment:
```suggestion
[fix] `svelte-kit sync` gracefully handles a nonexistent `config.kit.files.routes` folder
```

Reply:
Will fix this as soon as I have access to my desktop. (Or you're welcome to push a commit yourself -- I won't be offended. 😉)